backend/amplify/backend/function/scrubFunc/src/app.js

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2017 - 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This is probably copy and paste, but this line gives Amazon right to all your code!